sig-cloud-provider/2020-annual-report.md

```diff
@@ -0,0 +1,85 @@
+# 2020 - SIG Cloud Provider - Community Meeting Annual Reports
+---
+## Operational
+**How are you doing with operational tasks in sig-governance.md?**
+
+**Is your README accurate? have a CONTRIBUTING.md file?**
+
+Our README is accurate but we do not currently have a contributing.md file.
+
+**All subprojects correctly mapped and listed in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml)?**
+
+I believe so.
+
+**What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?**
+
+Our meetings tends to be small and quiet. We try to keep the meeting notes upto date during the meeting. 
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+_Same question as above but for working groups._
+
+We go ove the sub project updates during our primary biweekly meeting. 
+I believe the OWNERS files are all upto date.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+---
+## Membership
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
```

Comment:
Where are subproject owners listed?

Reply:
I believe they are listed in https://github.com/kubernetes/community/blob/master/sigs.yaml. These are mostly then redirects to other owners files. There is also https://github.com/kubernetes/community/blob/master/OWNERS_ALIASES but non of our sub-projects are listed there.